src/core/api/harborapi_test.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/goharbor/harbor/src/core/config"
 	"github.com/goharbor/harbor/src/core/filter"
 	"github.com/goharbor/harbor/tests/apitests/apilib"
+
```

Comment:
`goimports` did this, I can undo it if you want to keep the diff clean.

Reply:
No relationship with this PR, you can undo it.